pkgs/by-name/sk/skypilot/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchFromGitHub,
+  python3Packages
+}:
+
+python3Packages.buildPythonPackage rec {
```

Comment:
I haven't looked too deeply at the source, but it seems like the package you're building is only used purely as a binary, in which case this should be `buildPythonApplication` instead. If it's the case that you plan on importing the module, then you can keep as is.

See: https://nixos.org/manual/nixpkgs/unstable/#buildpythonapplication-function

Reply:
Good point and thanks for the feedback @aos - in its current form, it really is an application.

I will modify as you suggest.

Reply:
That's done now.